src/Property.ts

```diff
@@ -131,6 +133,30 @@ export default class Property<T = any> {
 		return String(def);
 	}
 
+	getColor(def?: T) {
+
+		const colorString = this.getString(def);
+
+		if (this.isCleanColor) {
+			return colorString;
+		}
+
+		this.isCleanColor = true;
+
+		if (!colorString.startsWith('rgb')) {
+			return colorString;
+		}
+
+		const cleanColorString = colorString.replace(
+			/(\d+)\.\d+\s*,/g,
```

Comment:
this just removes decimals? maybe another idea is just to do:
`color.r = parseInt(color.r, 10)` (and same for `g` and `b`) before we do `color.RGBA()`

but it does the job so i'll approve

Reply:
@gabelerner `rgbcolor` also can't parse rgb color like this `rgb(123.123, 23.22, 444.2)`, so this way is simpler